app/helpers/routes_helper.rb

```diff
@@ -67,4 +67,11 @@ def terms_path
       placements: "#",
     }.fetch HostingEnvironment.current_service(request)
   end
+
+  def guidance_path
+    {
+      claims: claims_grant_conditions_path,
+      placements: "#",
```

Comment:
Another domain related question (sorry), will the grant conditions need to be accessed from the placements side of the app?

Reply:
No, should be fixed by this https://github.com/DFE-Digital/itt-mentor-services/pull/394/files#r1535466462